app/src/androidTest/java/ch/epfl/sweng/SDP/utils/ImageStorageManagerTest.java

```diff
@@ -0,0 +1,60 @@
+package ch.epfl.sweng.SDP.utils;
+
+import android.Manifest;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.os.Environment;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.rule.GrantPermissionRule;
+import android.support.v4.app.ActivityCompat;
+
+import ch.epfl.sweng.SDP.R;
+import ch.epfl.sweng.SDP.auth.Account;
+import ch.epfl.sweng.SDP.auth.ConstantsWrapper;
+import ch.epfl.sweng.SDP.game.VotingPageActivity;
+import ch.epfl.sweng.SDP.game.drawing.DrawingOffline;
+import ch.epfl.sweng.SDP.home.HomeActivity;
+
+import java.io.File;
+
+import static ch.epfl.sweng.SDP.game.drawing.DrawingOnlineTest.initializedBitmap;
+import static ch.epfl.sweng.SDP.utils.ImageStorageManager.writeFileToStorage;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.Matchers.nullValue;
+import static org.junit.Assert.*;
+
+import org.junit.Rule;
+import org.junit.Test;
+
+public class ImageStorageManagerTest {
+
+    @Rule public GrantPermissionRule writeExternalStoragePermission =
```

Comment:
We always used the annotation in the line before the method definition, I think we should to the same here and below

Reply:
True